### PR TITLE
{lang} [intel/2018a] PyCUDA v2017.1.1 (REVIEW)

### PR DESCRIPTION
**Pull request by SimonPinches**

### CODE REVIEW

File: easybuild/easyconfigs/p/PyCUDA/PyCUDA-2017.1.1-intel-2018a-Python-2.7.14.eb [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+toolchain = {'name': 'intel', 'version': '2018a'}
+
+source_urls = [PYPI_LOWER_SOURCE]
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['6b5a9384e38c603ee429f8a6bee424532db7b3505027ce22f7e18ad19564b563']
+
+dependencies = [
+    ('Python', '2.7.14'),
+    ('CUDA', '9.1.85', '', True),
+    ('Boost', '1.67.0', versionsuffix),
```
Comment (boegel) [inline easybuild/easyconfigs/p/PyCUDA/PyCUDA-2017.1.1-intel-2018a-Python-2.7.14.eb]: @SimonPinches For easyconfigs using a `2018a` toolchain, we should stick to `Boost` 1.66.0` for now, to ensure that this module is compatible with others that depend on `Boost` (cfr. failing test)...

**Review by boegel — CHANGES_REQUESTED** [review]

**Comment by vanzod** [comment]
@SimonPinches I would recommend replacing the `Boost-1.66.0-Python-2.7.14` dependency with two separate dependencies as follows if the `Boost` Python bindings are needed:
```
('Boost', '1.66.0'),
('Boost.Python', '1.66.0', versionsuffix)
```

**Comment by boegel** [comment]
@vanzod Actually, just `Boost.Python` is enough (since `Boost` is a dep for `Boost.Python`).

I'll fix that in a bit...

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3102.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/6d6d52d1cc931da1c402a925c1cb6f71 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2006.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/2400379ecc2d8b94f3378d0171dc82b0 for a full test report.

**Review by boegel — APPROVED** [review]

**Comment by vanzod** [comment]
Test report by @vanzod
**SUCCESS**
Build succeeded for 2 out of 2 (1 easyconfigs in this PR)
cermis - Linux debian 9.9, Intel(R) Xeon(R) CPU           X5670  @ 2.93GHz, Python 2.7.13
See https://gist.github.com/a301984c23945d79fa420e80052bb6b0 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @SimonPinches!

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (1 easyconfigs in this PR)
node2434.golett.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/949c044b10156da98a20007695fa9bcb for a full test report.

**Pull request merged**